tests/utils/test_feature_utils.py

```diff
@@ -56,73 +57,51 @@ def test_convert_to_homogenous_coordinates_incorrect_dim(self):
         non_homogenous_coordinates = np.random.rand(5, 3)
 
         self.assertRaises(
-            TypeError,
-            feature_utils.convert_to_homogenous_coordinates,
-            non_homogenous_coordinates,
+            TypeError, feature_utils.convert_to_homogenous_coordinates, non_homogenous_coordinates,
         )
 
     def test_convert_to_epipolar_lines_valid_input(self):
-        """Test conversion of valid 2D points to epipolar lines using the essential matrix."""
-
-        points = np.array(
-            [
-                [10.0, -5.0],
-                [3.5, 20.0],
-            ]
-        )  # 2d points in homogenous coordinates
-        essential_mat = EssentialMatrix(Rot3.RzRyRx(0, np.deg2rad(45), 0), Unit3(np.array([-5, 2, 0])))
-
-        computed = feature_utils.convert_to_epipolar_lines(points, essential_mat)
-
-        expected = np.array(
-            [
-                [-2.36351579, -5.90878948, 1.75364193],
-                [-0.65653216, -1.64133041, -19.75129171],
-            ]
-        )
+        """Test conversion of valid 2D points to epipolar lines using the fundamental matrix, against manual
+        computation and with OpenCV's output."""
 
-        np.testing.assert_allclose(computed, expected)
+        points = np.array([[10.0, -5.0], [3.5, 20.0],])  # 2d points in homogenous coordinates
+        E_matrix = EssentialMatrix(Rot3.RzRyRx(0, np.deg2rad(45), 0), Unit3(np.array([-5, 2, 0])))
+        F_matrix = E_matrix.matrix()  # using identity intrinsics
+        expected_opencv = cv.computeCorrespondEpilines(points.reshape(-1, 1, 2), 1, F_matrix)
```

Comment:
can we put expected output shape in inline comment?

Reply:
added a node where i use `np.squeeze`